code/game/objects/items/devices/megaphone.dm

```diff
@@ -19,7 +19,7 @@
 
 /obj/item/megaphone/equipped(mob/M, slot)
 	. = ..()
-	if (slot == ITEM_SLOT_HANDS)
+	if (slot == ITEM_SLOT_HANDS && !HAS_TRAIT(M, TRAIT_SIGN_LANG))
```

Comment:
Is there some way to fenangle the signals to make this cleaner? It's fine if not, this just bothers me.

Reply:
If there is I don't know of it at the moment, however once I (hopefully) have more time later today, I'll look deeper into it.